Commands/MessageCount/pagination.go

```diff
@@ -0,0 +1,45 @@
+{{/*
+	This is the pagination code for the message leaderboard
+  
+	Recommended trigger: added reactions only
+	Trigger type: reaction
+*/}}
+{{/* dont touch */}}
+{{ $action := .Reaction.Emoji.Name }} {{/* The action being ran */}}
+{{ $validEmojis := cslice "➡" "⬅" }} {{/* Valid emojis */}}
+{{ $isValid := false }} {{/* Whether this is actually a valid embed / leaderboard embed */}}
+{{ $page := 1 }} {{/* The current page */}}
+{{ with and (eq .ReactionMessage.Author.ID 204255221017214977) .ReactionMessage.Embeds }} {{/* Checks for validity */}}
+	{{ $embed := index . 0 }} {{/* The first embed */}}
+	{{ if and (eq $embed.Title (print "❯ Messages leaderboard")) $embed.Footer }} {{/* More checks */}}
+		{{ $page = reFind `\d+` $embed.Footer.Text }} {{/* We presume that this is valid, and get the page num */}}
+		{{ $isValid = true }} {{/* Yay, it is valid */}}
+		{{ deleteMessageReaction nil $.ReactionMessage.ID $.User.ID $action }}
+	{{ end }}
+{{ end }}
+{{ if and (in $validEmojis $action) $isValid $page }} {{/* Even more checks for validity... */}}
```

Comment:
too much validation gets chunky, only really need {{if and $page (in $validEmojis $action)}} or if you wanna use pipelines (although increase unreadability) via {{if in $validEmojis $action|and $page}}